apps/nextjs-website/src/components/Header.tsx

```diff
@@ -2,9 +2,52 @@
 import React from 'react';
 import { Header as HeaderEC } from '@pagopa/pagopa-editorial-components/dist/components/Header';
 import { HeaderProps } from '@pagopa/pagopa-editorial-components/dist/components/Header/Header';
+import { Stack } from '@mui/material';
+import { usePathname } from 'next/navigation';
 
-const Header: React.FC<HeaderProps> = (headerData) => (
-  <HeaderEC {...headerData} />
-);
+const Header: React.FC<HeaderProps> = (headerData) => {
```

Comment:
Even though [it has been fixed now](https://www.totaltypescript.com/you-can-stop-hating-react-fc), until recently, it was [discouraged](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/#:~:text=With%20latest%20React%20types%20and%20TypeScript%205.1.%20it%27s%20mostly%20a%20stylistic%20choice%2C%20otherwise%20discouraged.). I would avoid the `React.FC` form and write it as follows: 

```suggestion
const Header = (props: HeaderProps) => {
```

>  [!WARNING]
> I have renamed the parameters, so if you accept this suggestion, you need to rename them in the body of the function.